Observer/SubmitQuoteObserver.php

```diff
@@ -25,7 +25,12 @@ public function execute(Observer $observer)
         }
 
         // Further shopper action required (e.g. redirect or 3DS authentication)
-        if (in_array($payment->getMethod(), ['adyen_hpp', 'adyen_cc', 'adyen_oneclick'], true)) {
+        //TODO: Once we have a config in the magento backoffice, get all the methods directly from this config
+        if (in_array(
+            $payment->getMethod(),
+            ['adyen_hpp', 'adyen_cc', 'adyen_oneclick', 'adyen_paypal', 'adyen_ideal'],
```

Comment:
That observer still listens to `adyen_hpp` payment method to keep the cart active if additional step is required to complete the payment. Do we need to update this method with `getAdyenPaymentMethods()` function? Is this observer still valid?